### PR TITLE
Add format step

### PR DESCRIPTION
**Pull request by streeve**

### CODE REVIEW

File: .github/workflows/CI2.yml [hunk 1]
```diff
@@ -127,5 +127,10 @@ jobs:
           cmake --build build --parallel 2
           CTEST_OUTPUT_ON_FAILURE=1 cmake --build build --target test
           cmake --install build
+      - name: Format Cabana
```
Comment (sslattery) [inline .github/workflows/CI2.yml:130]: Curious if this will find clang format 10

Comment (streeve) [reply .github/workflows/CI2.yml:130]: We're getting the default: https://github.com/kokkos/ci-containers/blob/071c72d137a9fa8f203dc5bc2b4572be8cc210cc/ubuntu#L8 

Which looks to be working

**Review by sslattery — APPROVED** [review]

File: .github/workflows/CI2.yml [hunk 2]
```diff
@@ -127,5 +127,11 @@ jobs:
...
+        if: ${{ matrix.distro == 'ubuntu:latest' }}
+        run: |
+             cd build
```
Comment (junghans) [inline .github/workflows/CI2.yml:133]: I would have used `working-directory: build` instead.

**Comment by streeve** [comment]
#345

**Pull request merged**